lib/presentation/widget/myspace/my_space_widget.dart

```diff
@@ -73,17 +73,12 @@ class _MySpaceWidgetState extends State<MySpaceWidget> {
         builder: (BuildContext context, MySpaceViewModel viewModel) => Scaffold(
               body: Column(
                 children: [
-                  StreamBuilder(builder: (context, snapshot) {
```

Comment:
@manhduc-nguyen don't think StreamBuilder is needed here because viewState is not a stream, can you confirm ? 
We have to specify a `stream `
```dart
       child: StreamBuilder<int>(
          stream: _bids,
          builder: (BuildContext context, AsyncSnapshot<int> snapshot) {
```

Reply:
Yes, We only need `StoreConnector` to do this. Wrong understanding when setup `Redux`. Please remove `StreamBuilder` here